LoggerApp/Logger.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LoggerApp
+{
+    public sealed class Logger
+    {
+        private readonly DateTime _dateOfLog = DateTime.Now;
+        private string _logType;
+        private string _message;
+
+        private static readonly Logger instance = new Logger();
+        private List<object> _logs = new List<object>();
+
+        public static Logger Instance
+        {
+            get => instance;
+        }
+
+        static Logger() {}
+
+        private Logger() {}
+
+        private Logger(DateTime dateOfLog, string logType, string message)
+        {
+            _dateOfLog = dateOfLog;
+            _logType = logType;
+            _message = message;
+        }
+
+        public void AddLogData(string logType, string message)
+        {
+            _logs.Add(new Logger(_dateOfLog, logType, message));
```

Comment:
You are doing something wrong here. We need to store logs. The log should be represented with a separate class or structure (in HW it is named Result). Here you store Loggers in the Loggers list. With all unnecessary information.